Gemfile

```diff
@@ -10,6 +10,10 @@ def location_for(place, fake_version = nil)
   end
 end
 
+# The public_suffix gem used in the dependencies for the packaging gem no longer supports Ruby 2.5
+# and bundler is not smart enough to realize that an older version would work in the mix.
+gem 'public_suffix', '= 4.0.7'
+
```

Comment:
(nit) can this go under the 'packaging' entry to emphasize that this is a dependency of the packaging gem and not something that we need otherwise? I know the comment mentions it, but the grouping would help my brain. ¯\_(ツ)_/¯

Reply:
Doesn't the install need to be _before_ the packaging gem in order for the packaging install not to fail?

Reply:
oh hmm maybe. I guess I haven't looked into how exactly Gemfiles are evaluated. In that case, can we just remove the extra newline that separates this from the 'packaging' entry?

Reply:
Sure thing, done.